charts/values.yaml.changelog.md

```diff
@@ -5,6 +5,12 @@ Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
 ----
+
+## Changes on top of Renku 0.7.0
```

Comment:
```suggestion
## Upgrading to Renku 0.8.0 (INCLUDES BREAKING CHANGES!)
```
I guess this should go into a `0.8.0` release as it requires changes to the values file, right?

Reply:
I thought the version 0.7.0 is yet to be released, right? Should I move these changes in the already existing 0.7.0 section?

Reply:
ah yeah, ofc, sorry. Yes, just add it to the 0.7.0 section 👍